ostd/src/arch/loongarch/mm/mod.rs

```diff
@@ -205,24 +185,24 @@ impl PageTableEntryTrait for PageTableEntry {
     }
 
     fn prop(&self) -> PageProperty {
-        let flags = (parse_flags!(!(self.0), PageTableFlags::NOT_READABLE, PageFlags::R))
-            | (parse_flags!(self.0, PageTableFlags::WRITABLE, PageFlags::W))
-            | (parse_flags!(!(self.0), PageTableFlags::NOT_EXECUTABLE, PageFlags::X))
+        let flags = parse_flags!(!(self.0), PteFlags::NOT_READABLE, PageFlags::R)
+            | parse_flags!(self.0, PteFlags::WRITABLE, PageFlags::W)
+            | parse_flags!(!(self.0), PteFlags::NOT_EXECUTABLE, PageFlags::X)
             // TODO: How to get the accessed bit in loongarch?
-            | (parse_flags!(self.0, PageTableFlags::PRESENT, PageFlags::ACCESSED))
-            | (parse_flags!(self.0, PageTableFlags::DIRTY, PageFlags::DIRTY))
-            | (parse_flags!(self.0, PageTableFlags::RSV2, PageFlags::AVAIL2));
-        let mut priv_flags = parse_flags!(self.0, PageTableFlags::RSV1, PrivFlags::AVAIL1);
+            | parse_flags!(self.0, PteFlags::PRESENT, PageFlags::ACCESSED)
+            | parse_flags!(self.0, PteFlags::DIRTY, PageFlags::DIRTY)
+            | parse_flags!(self.0, PteFlags::RSV2, PageFlags::AVAIL2);
+        let mut priv_flags = parse_flags!(self.0, PteFlags::RSV1, PrivFlags::AVAIL1);
```

Comment:
Insert a blank line before `let priv_flags` since we should now separate `flags`, `priv_flags`, and `cache` into different code blocks (as done in other architectures), and `cache` is already in a separate code block below.